### ISSUE TITLE
This dependency was not found:  * !!vue-style-loader!css-loader?{

### ISSUE THREAD

**Issue by PecoKael** [surmon-china/vue-awesome-swiper#104]
This dependency was not found:

* !!vue-style-loader!css-loader?{"minimize":false,"sourceMap":false}!../._vue-loader@12.2.1@vue-loader/lib/style-compiler/index?{"vue":true,"id":"data-v-6d1be154","scoped":false,"hasInlineConfig":false}!node_modules/swiper/dist/css/swiper.css in ./~/vue-awesome-swiper/swiper.vue

To install it, you can run: npm install --save !!vue-style-loader!css-loader?{"minimize":false,"sourceMap":false}!../._vue-loader@12.2.1@vue-loader/lib/style-compiler/index?{"vue":true,"id":"data-v-6d1be154","scoped":false,"hasInlineConfig":false}!node_modules/swiper/dist/css/swiper.css

**Comment by surmon-china** [comment]
#41

**Comment by a459295360** [comment]
you must install sass-loader with npm or yarn

**Comment by HaroldV** [comment]
Thanks @a459295360 :)

**Comment by lau-a** [comment]
why isn't this installed by default?

**Comment by TurboTobias** [comment]
@lau-a Agree. When initializing a project with the CLI, the question "add a css pre-processor [scss, less, stylus]?" would be nice

**Comment by exabyter** [comment]
Thank @a459295360

**Comment by lazycece** [comment]
to excute “npm install node-sass sass-loader” can do,  but why there was  error "!vue-style-loader!css-loader"?

**Comment by MTTTM** [comment]
PS G:\my-project> cnpm i vue-style-loader css-loader
√ Installed 2 packages
√ Linked 0 latest versions
√ Run 0 scripts
√ All packages installed (used 208ms, speed 17.6kB/s, json 2(3.66kB), tarball 0B)
PS G:\my-project> npm run dev

> x@1.0.0 dev G:\3k\my-project
> webpack-dev-server --inline --progress --config build/webpack.dev.conf.js

 94% asset optimization

 ERROR  Failed to compile with 1 errors                                                                                                              18:19:04

This dependency was not found:

* !!vue-style-loader!css-loader?{"sourceMap":true}!../../node_modules/._vue-loader@13.7.1@vue-loader/lib/style-compiler/index?{"vue":true,"id":"data-v-656039f0","scoped":true,"hasInlineConfig":false}!less-loader?{"sourceMap":true}!../../node_modules/._vue-loader@13.7.1@vue-loader/lib/selector?type=styles&index=0!./HelloWorld.vue in ./src/components/HelloWorld.vue

To install it, you can run: npm install --save !!vue-style-loader!css-loader?{"sourceMap":true}!../../node_modules/._vue-loader@13.7.1@vue-loader/lib/style-compiler/index?{"vue":true,"id":"data-v-656039f0","scoped":true,"hasInlineConfig":false}!less-loader?{"sourceMap":true}!../../node_modules/._vue-loader@13.7.1@vue-loader/lib/selector?type=styles&index=0!./HelloWorld.vue


Install all，but it also throws Error.

**Comment by joshuahiskraken** [comment]
make sure the directory names don't contain '!' it will throw this error

**Comment by danicholls** [comment]
This helped me: https://github.com/nuxt/nuxt.js/issues/1069#issuecomment-314422716

**Comment by ultravires** [comment]
hey, if you want to use `stylus`, please:
>npm install stylus stylus-loader --save-dev

**Comment by anson-GH** [comment]
reinstall
npm uninstall sass-loader 
npm install sass-loader@7.3.1 --save-dev

add below in your webpack
![Screenshot_112319_120628_AM](https://user-images.githubusercontent.com/15632074/69441140-19a0c700-0d85-11ea-9555-38f082665599.jpg)

**Comment by nicely** [comment]
> I got error in version sass-loader: 8.x
> 
> I do reinstall and work for me.
> npm uninstall sass-loader
> npm install sass-loader@7.3.1 --save-dev
> 
> add below in your webpack
> ![Screenshot_112319_120628_AM](https://user-images.githubusercontent.com/15632074/69441140-19a0c700-0d85-11ea-9555-38f082665599.jpg)

Thank you.... I hate those NPM versions........ hateeeeeee but still using...

**Comment by rsbwdgzq** [comment]
It still not work...
